src/main/java/hudson/plugins/cobertura/CoberturaPublisher.java

```diff
@@ -889,6 +891,23 @@ public ParseReportCallable(String reportFilePath) {
             CoverageMetric.LINE,
             CoverageMetric.CONDITIONAL,};
 
+        /**
+         * load global config from disk
+         */
+        public DescriptorImpl(Class<? extends Publisher> clazz) {
```

Comment:
Is it actually used anywhere?

Reply:
The comment of `Descriptor` said `Descriptor can persist data just by storing them in fields. However, it is the responsibility of the derived type to properly invoke save() and load().` 
So I call `save()` in `configure()` to persist global configuration. And the comment of `load()` said `The constructor of the derived class must call this method. (If we do that in the base class, the derived class won't get a chance to set default values.)`. So I follow the comment to implemented its two constructors and call load method.

Reply:
It provides a config setting on the plugin page:

<img width="328" alt="screenshot 2018-03-15 07 36 24" src="https://user-images.githubusercontent.com/1479145/37470052-e2d9b21a-2823-11e8-9249-77325ac490e6.png">